### PR TITLE
add survey notice and image

### PR DESCRIPTION
**Pull request by matthewbirtch**
### In-product notice for user feedback survey
- Adds a notice and supporting image for user feedback survey
- Main call to action links to google form: https://forms.gle/h7SnAehdzbDb83x78

Conditions:
- Display for cloud and on-prem
- Display for servers >=7.1
- Display for desktop and browser clients

This is a notice similar to that which was posted last year in: https://github.com/mattermost/notices/pull/292

### CODE REVIEW

File: notices.json [hunk 1]
```diff
@@ -1,4 +1,22 @@
 [
+  {
+    "id": "user_survey_2023",
+    "conditions": {
+      "displayDate": "<= 2023-11-30T00:00:00Z",
```
Comment (amyblais) [inline notices.json:5]:
```suggestion
      "displayDate": "<=2023-11-30T00:00:00Z",
```

**Review by amyblais — APPROVED** [review]

**Review by yasserfaraazkhan — DISMISSED** [review]
Screenshot 2023-10-16 at 3 50 44 PM

Screenshot 2023-10-16 at 3 50 53 PM

image

**Comment by matthewbirtch** [comment]
@amyblais @yasserfaraazkhan sorry, I realized a minor error in the text and had to update. Do you mind approving again?

**Comment by matthewbirtch** [comment]
@wiersgallak or @esethna the readme for the notices repo says only PMs can merge to release branch. Could you approve and merge so the survey notice can begin to go out?

**Comment by esethna** [comment]
@matthewbirtch @cwarnermm before this goes out, have we updated the docs and in-product text to expand the purpose of these notices to include gathering user feedback to improve the product?

**Comment by cwarnermm** [comment]
Thanks, @esethna. That's next on my list! In fact, I'll get a docs PR started now to incorporate your feedback and avoid delaying this PR from moving forward.

**Comment by cwarnermm** [comment]
In-product text updates: https://github.com/mattermost/mattermost/pull/25004

**Comment by cwarnermm** [comment]
Config setting docs updates: https://github.com/mattermost/docs/pull/6726

**Comment by matthewbirtch** [comment]
Thanks for tackling this so quickly @cwarnermm! It's much appreciated

**Comment by esethna** [comment]
@cwarnermm let me know once those all are merged and then we can ship this notice. Thanks!

**Review by wiersgallak — APPROVED** [review]
LGTM

**Comment by cwarnermm** [comment]
@esethna - Supporting documentation and in-product text updates are merged into their respective code bases.

**Comment by esethna** [comment]
@matthewbirtch merged

**Pull request merged**